app/main.py

```diff
@@ -1,2 +1,92 @@
+from typing import Any
+
+
 class Dictionary:
-    pass
+    def __init__(
+            self,
+            initial_capacity: int = 8,
+            load_factor: float = 0.66) -> None:
+        self.initial_capacity = initial_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.capacity = initial_capacity
+        self.table = [None] * initial_capacity
+
+    def __setitem__(self, key: Any, value: Any) -> None:
```

Comment:
The key should be hashable. Indicate it in the type annotation accordingly throughout the code.